src/proj_4D_api.c

```diff
@@ -480,9 +481,18 @@ Returns 1 on success, 0 on failure
         double *d = P->datum_params;
         size_t n = strlen (s);
 
+
         /* We ignore null helmert shifts (common in auto-translated resource files, e.g. epsg) */
         if (0==d[0] && 0==d[1] && 0==d[2] && 0==d[3] && 0==d[4] && 0==d[5] && 0==d[6])
+        {
+            /* If the current ellipsoid is not WGS84, then make sure frame */
```

Comment:
I suggest you change the comment to:

"If the current ellipsoid is not WGS84, then make sure the change in ellipsoid is still done". Since the Helmert transform is a no-op it is a little confusing to me to talk about a change in frame.

Reply:
Suggested comment adopted